corehq/util/couchdb_management.py

```diff
@@ -59,5 +65,11 @@ def get_db_for_class(self, klass):
     def get_db_for_doc_type(self, doc_type):
         return Database(self.get_db_uri_for_doc_type(doc_type))
 
+    def get_db_for_postfix(self, postfix):
+        return Database(self.all_db_uris_by_slug[postfix])
```

Comment:
I think this function is exactly the same as `get_db` above, except it doesn't use `create=True`. The name for `get_db` doesn't follow my own naming convention; I named it like that as an analogy for the global `get_db(slug)` function we use everywhere else.

So a couple things. (1) Are you intentionally adding this function to not use `create=True`? and (2) if you want to add this function, can we call it `get_db_for_slug`?